reporters/reporter-default/src/main/java/org/radargun/reporting/csv/CsvReporter.java

```diff
@@ -287,4 +325,95 @@ private double round(double number, int places) {
       return bigDecimal.doubleValue();
    }
 
+   /**
+    * Comparator for sorting rows based on column value
+    * 
+    * @author zhostasa
```

Comment:
It was decided in https://github.com/radargun/radargun/pull/385 to remove the author tags, so I think no new ones should be added.